client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -909,6 +909,17 @@ Params withWaitForActiveShards(ActiveShardCount activeShardCount) {
             return withWaitForActiveShards(activeShardCount, ActiveShardCount.DEFAULT);
         }
 
+        /**
+         * @deprecated <code>copy_settings</code> defaults to <code>true</code> and can not be set to false
```

Comment:
Note that the default is unset and behaves as `false`, however `false` can not be set explicitly. `copy_settings` is either unset (default) or set to `true` (future behavior).

Reply:
Thanks great catch this got forgotten while I was looking the behavior of other branches.
Addressed in https://github.com/elastic/elasticsearch/pull/39752/commits/ce68119b9315a4dc812146e9e40b29cb8f9344f1